src/Annotation/Query.php

```diff
@@ -12,10 +12,16 @@
  */
 final class Query extends Field
 {
+    /**
+     * @deprecated
+     * @var array<string>
```

Comment:
```suggestion
     * @var array<string>
     *
     * @deprecated This property is deprecated since 1.0 and will be removed in 1.1. Use $targetTypes instead.
```